### ISSUE TITLE
How's the project coming along?

### ISSUE THREAD

**Issue by petonic** [hunternet93/dymo-web-service-clone#1]
Hi -- I'm interested in printing to my Dymo PnP LabelPrinter hooked to the USB of my Raspberry Pi.

How's the project going?

**Comment by hunternet93** [comment]
I haven't done much on it, my church was considering switching to a check-in system that used the Dymo API for printing but we ended up staying with our current provider.

The project would be pretty simple though, all the Dymo service does it start up a simple HTTPS server that responds to a few API calls. The difficulty lies in figuring out how to render Dymo's custom label XML, for my application I planned to just make my own templates to print the data instead but obviously that wouldn't be ideal for most uses.

If you're interested I could upload my notes on the API, and possibly some proof-of-concept server code.

**Comment by mcarter00** [comment]
If you want to upload what you've got, I'd love to help.

**Comment by hunternet93** [comment]
Sorry for the very late response. I still haven't worked on this at all, but [here's what I found](https://gist.github.com/hunternet93/05592a8fa07987617c3043f00aeddf33) when I took a look at it a few months back.

With Dymo's software installed on a Windows machine, it's pretty easy to see what's going on just by watching in a browser's dev tools while printing labels. Coding up a FOSS implementation should be pretty simple, unfortunately I have several other projects going on at the moment.
